github/git_blobs.go

```diff
@@ -38,6 +39,23 @@ func (s *GitService) GetBlob(ctx context.Context, owner string, repo string, sha
 	return blob, resp, err
 }
 
+// GetBlobRaw fetches a blob's contents from a repo.
+// Unlike GetBlob, it returns the raw bytes rather than the base64-encoded data.
+func (s *GitService) GetBlobRaw(ctx context.Context, owner, repo, sha1 string) ([]byte, error) {
+	u := fmt.Sprintf("repos/%v/%v/git/blobs/%v", owner, repo, sha1)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Accept", "application/vnd.github.v3.raw")
+
+	var buf bytes.Buffer
+	if _, err = s.client.Do(ctx, req, &buf); err != nil {
```

Comment:
I would suggest having `_, err = s.client.Do(ctx, req, &buf)` be on a separate line, like this:

```Go
_, err = s.client.Do(ctx, req, &buf)
if err != nil {
    ...
}
```

Making it inline in the `if` statement makes it look less important, and it's not consistent with the style in the hundreds of other endpoints we have. I think it's a good idea to try to keep the code in this library as consistent as possible, and not deviate unnecessarily. Inconsistent style makes it harder for new contributors to figure out which style to use.

Reply:
seems like unnecessary consistency, but sure.